### DISCUSSION TITLE
Any tips to save archived photos ?!

### DISCUSSION THREAD

**Discussion by DraGula** [gilesknap/gphotos-sync#489] [Q&A]
Hi there,

I've tested several settings, and I can't find my archived photos (by searching via the .jpg file name).
I missed something to make a backup of my archived google photos?
I've also tested with argument : --archived
idem. I visibly have all my photos out of the archives...

Am I missing something?
Thanks!

**Comment by sebrem** [comment]
I can't confirm this, I just archived photos before downloading with `--archive`  and they're in the `yyyy/dd` folder next to the others. Maybe you made a typo `--archived` instead of `--archive`?

edit: I just see the help indeed says `--archived` but for me it does work with `--archive` :exploding_head:  (version: 3.2.2, database schema version 5.7)

**Comment by DraGula** [comment]
I also tested --archive
Same result
And yet in the logs I see "includeArchivedMedia': True".

Version: 3.2.2
Database schema version 5.7

No idea...

**Comment by gilesknap** [comment]
Hi @DraGula as far as my testing goes, the --archive option works. I have not tried it on my own library for a few years now, but there is a system test that checks its function and that ran recently.
https://github.com/gilesknap/gphotos-sync/blob/219d2852d9e72ce58485d99e6b8db4eef1f9e9ee/tests/test_system/test_system.py#L60-L75

Sorry about the mis-spelling of the option in the help!

I don't know what to suggest - there are a few bugs in the Google API that have been long reported and some of them relate to failing to list some photos when getting the index. There don't seem to be any fixes coming soon :-(
See #119

**Comment by DraGula** [comment]
Hi @gilesknap 
Thanks for your feedback. Very clear
I'll follow the Google feed, maybe the api will be corrected one day :)
The main thing is that the rest works perfectly.
Thanks to you for the job !!
